pages/account.js

```diff
@@ -13,9 +13,14 @@ const AccountPage = ({ url, me, t }) => {
   }
   return (
     <Frame url={url} meta={meta}>
-      {me && me.id ? <Account /> : <Marketing />}
+      <Account />
     </Frame>
   )
 }
 
-export default compose(withData, withMe, withT)(AccountPage)
+export default compose(
+  withData,
+  withAuthorization(['admin']),
```

Comment:
no role here: you can always manage you account even with an expired membership. Extend `withAutorization` to allow usage with no `authorizedRoles` (undefined) and simply do the me or login screen check.